dsperse/src/server/handlers.py

```
@@ -0,0 +1,388 @@
+
+        run_output_dir = Path(tmp_dir) / "run"
+        runner = Runner(
+            run_dir=str(run_output_dir),
+            on_slice_complete=on_slice_complete,
+        )
+        try:
+            runner.run(
+                input_json_path=str(input_path),
+                slice_path=str(slices_dir),
+            )
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)